src/sysinfo/nvapi_adapter.cpp

```diff
@@ -120,9 +130,15 @@ namespace dxvk {
 
             nvmlDevice_t nvmlDevice{};
             auto result = m_nvml.DeviceGetHandleByPciBusId_v2(pciId, &nvmlDevice);
-            if (result == NVML_SUCCESS)
+            if (result == NVML_SUCCESS) {
                 m_nvmlDevice = nvmlDevice;
-            else
+
+                nvmlMemory_v2_t memory{};
+                memory.version = 2;
```

Comment:
Are you sure the value should be just `2` and not `nvmlMemory_v2`? I made such a mistake while testing and it caused the call to return `NVML_FUNCTION_NOT_FOUND` for me :/

Reply:
You are right ofc, fixed and also validated the NVML path:

```    
Dedicated video memory:     24576MB
Available dedicated v.m.:   24131MB
System video memory:        0MB
Shared system memory:       48119MB
Cur. avail. dedicated v.m.: 23787MB
```

The documentation tricked me ;) https://docs.nvidia.com/deploy/nvml-api/structnvmlMemory__v2__t.html#structnvmlMemory__v2__t states  `Structure format version (must be 2).`